Resources/engineCommandPerms.yml

```diff
@@ -91,6 +91,8 @@
   - tp
   - tpto
   - respawn
+  - loadgrid
+  - tpgrid
```

Comment:
This category is to do with adminning players, not adminning the server. It belongs in the mapping category where it already is.